R/bootstrap.R

```diff
@@ -42,10 +42,37 @@ bootstrapPage <- function(..., title = NULL, responsive = NULL, theme = NULL) {
       # remainder of tags passed to the function
       list(...)
     ),
-    bootstrapLib()
+    if (useBsTheme()) bootstraplib::bootstrap() else bootstrapLib()
```

Comment:
Move this logic into `bootstrapLib()`

Reply:
Done in 61509ccbc55900bad795eb3d320f0adddb2dc298